src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+export const booksState = [];
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const booksReducer = (booksState, action) => {
```

Comment:
- [ ] Make sure that a new book is added to an array, so please initialize a `booksState` equal to an empty array.